mesheryctl/pkg/utils/platform.go

```diff
@@ -36,78 +36,6 @@ var (
 	ReleaseTag      string
 )
 
-// ChangePlatform changes the platform specified in the current context to the specified platform
-func ChangePlatform(currCtx string, ctx config.Context) error {
```

Comment:
Gooood.

But instead of the new function, let's consolidate to using the struct functions.